clients/cobol-lsp-vscode-extension/src/services/ProcessorGroups.ts

```diff
@@ -51,26 +58,49 @@ export async function loadProcessorGroupCopybookPaths(
 export async function loadProcessorGroupCopybookPathsConfig(
   item: { scopeUri: string },
   configObject: string[],
-): Promise<string[]> {
-  const cfg = await loadProcessorGroupSettings(
-    item.scopeUri,
-    "libs",
-    [] as string[],
-  );
-  const config = SettingsService.evaluateVariables(
-    [...cfg, ...configObject],
-    getVariablesFromUri(item.scopeUri, false),
-  );
+  dialect?: string,
+): Promise<
+  (string | ZoweDatasetConfigModel | ZoweUssConfigModel | EndevorConfigModel)[]
+> {
+  const allConfigs = [
+    ...(await loadProcessorGroupSettings(
+      item.scopeUri,
+      "libs",
+      [] as string[],
+      dialect,
+    )),
+    ...configObject,
+  ];
 
-  const wsUri = workspace.getWorkspaceFolder(Uri.parse(item.scopeUri))?.uri;
-  if (wsUri === undefined) {
-    return configObject;
+  const configs: (
+    | string
+    | ZoweDatasetConfigModel
+    | ZoweUssConfigModel
+    | EndevorConfigModel
+  )[] = [];
+
+  for (const config of allConfigs) {
+    if (typeof config === "string") {
+      const tempConfig = SettingsService.evaluateVariables(
+        [config],
+        getVariablesFromUri(item.scopeUri, false),
+      );
+
+      const wsUri = workspace.getWorkspaceFolder(Uri.parse(item.scopeUri))?.uri;
+      if (wsUri === undefined) {
+        configs.push(config);
+      } else {
+        const globs = globSync(
+          tempConfig.map((ele) => ele.replace(backwardSlashRegex, "/")),
+          { cwd: cleanWorkspaceFolderName(wsUri.fsPath), absolute: true },
+        ).map((s) => normalizePath(s));
+        configs.push(...globs);
+      }
+    } else {
+      configs.push(config);
```

Comment:
The other configurations should support variable substitutions as well.

Reply:
At minimum, various filename/dirname related variables have the same uses in uss as they have when specifying local paths.